Updated dependencies. Due to this reason there are so many changed fi…

impecablemee opened the pull request:
…les.

Some modules are not only updated, but replaced entirely. For example, `reconnecting-websocket` -> `socket.io`, as the last one support much more cool functionality.
Make the project work with new dependencies.
Sass pre-processor (.scss) enabled via webpack configs.
Aliases enabled.
Began to fix styles. Particularly, the left panel (consisting of users list, login and logged-in components) and the chat header  styles are changed to the adaptive ones.

impecablemee committed:
Updated dependencies. Due to this reason there are so many changed files.
Some modules are not only updated, but replaced entirely. For example, `reconnecting-websocket` -> `socket.io`, as the last one support much more cool functionality.
Make the project work with new dependencies.
Sass pre-processor (.scss) enabled via webpack configs.
Aliases enabled.
Began to fix styles. Particularly, the left panel (consisting of users list, login and logged-in components) and the chat header  styles are changed to the adaptive ones.

impecablemee committed:
week-2-3

impecablemee committed:
week-2-3
node_modules removed

impecablemee committed:
Delete .DS_Store

impecablemee committed:
Delete CHANGES.md~

impecablemee committed:
Delete README.md~

impecablemee committed:
Delete .DS_Store

impecablemee committed:
Delete README.md~

impecablemee committed:
Delete .DS_Store

impecablemee committed:
Delete .DS_Store

impecablemee committed:
Delete .History.md.un~

impecablemee committed:
Delete .Readme.md.un~

impecablemee committed:
Delete .package.json.un~

impecablemee committed:
Delete .index.js.un~

impecablemee committed:
Delete .DS_Store

impecablemee committed:
week-2-3
node_modules added to .gitignore

kos33rd reviewed (commented):
К сожалению, веб-интерфейс гитхаба абсолютно не приспособлен к просмотру такого большого количества изменённых файлов, поэтому ориентируюсь на то, что сейчас по факту находится в ветке `week-2-3`.

1. Пакет `concurrently` отсутствует в корневом `package.json`, из-за чего приложение не получится запустить "из коробки". Нужно добавить.
2. То же касается `nodemon` у сервера.

Для выполнения задания второй недели этих двух правок будет достаточно. Для третьей недели предлагаю смотреть код отдельно - либо в этом PR, либо в отдельном (можно даже без изменений кода - просто описать, что где в коде находится по заданию).
По заданиям третьей недели сейчас вижу, что не хватает роутинга.

impecablemee committed:
week-2-3
package.json fixed

impecablemee committed:
week-2-3
routing added: particularly, settings page added. The opportunity to change the interface language is going to be implemented

petrNikolaev1 commented:
@kos33rd Посмотрите, пожалуйста, исправления

kos33rd reviewed (approved):
Теперь всё ок, апрув 👌

AVVlasov commented on client/src/containers/Settings.js:
```diff
@@ -0,0 +1,18 @@
+export default class ChatWrap extends Component {
+    render() {
+        return (
+            <div>
```
не забываем что есть React.Fragment
